IdentityCore/src/workplacejoin/MSIDPkeyAuthHelper.m

```diff
@@ -140,8 +141,18 @@ + (NSString *)createDeviceAuthResponse:(NSString *)audience
         return nil;
     }
     NSArray *arrayOfStrings = @[[NSString stringWithFormat:@"%@", [[identity certificateData] base64EncodedStringWithOptions:0]]];
+#if MSID_ENABLE_ECC_SUPPORT
```

Comment:
Why do we need that #if block? I thought MSIDKeyOperationUtil already return correct algorithm for both ECC/RSA key?